containers/base/javascript/browser/css/dashboard.css

```diff
@@ -47,6 +83,93 @@ progress[value]::-moz-progress-bar {
   background-color: #391;
```

Comment:
This wasn't part of your changes, but use variables here as well for consistency?